Zend/zend_closures.c

```diff
@@ -147,7 +147,7 @@ ZEND_METHOD(Closure, call)
 
 	fci.retval = &closure_result;
 	fci.params = my_params;
-	fci.param_count = my_param_count;
+	fci.param_count = -my_param_count;
```

Comment:
Small note: negative param count is a hack to avoid copying around. I'm open for other suggestions. (let's discuss that later.)